hybrid/decomposers.py

```diff
@@ -422,6 +428,99 @@ def next(self, state, **runopts):
 
         return state.updated(subproblem=subbqm)
 
+class SublatticeDecomposer(traits.ProblemDecomposer, traits.SISO, Runnable):
+    """Selects a lattice-structured subproblem.
+
+    This decomposer requires the input state has fields ``bqm`` and 
+    ``origin_embeddings``, only the keys from ``origin_embeddings`` are used.
```

Comment:
"only the keys from ``origin_embeddings``" --> is it worth pointing out that keys means the variables? something like, "only the keys (the variables/nodes) from ``origin_embeddings`"